client/client.go

```diff
@@ -2,14 +2,75 @@ package client
 
 import (
 	"context"
+	"fmt"
+	"strings"
+	"time"
 
 	"github.com/mhemmings/triage/issues"
 )
 
+// IssueFilters contains the possible filters to use when searching for issues.
+type IssueFilters struct {
+	Labels  Labels
```

Comment:
Perhaps document when these flags are active. For example "ShowAll" feels like the opposite of a filter - it actually adds to the set of possible results AFAICS.